tests/tezt/_regressions/tests/OK/hott.dk.out

```diff
@@ -0,0 +1,50 @@
+tests/tezt/_regressions/tests/OK/hott.dk.out
```

Comment:
I think the command should be in the regression filename otherwise it might be easy to introduce a conflict.